core/src/main/java/jenkins/security/LastGrantedAuthoritiesProperty.java

```diff
@@ -148,10 +148,6 @@ protected void loggedOut(@Nonnull String username) {
 
     @Extension
     public static final class DescriptorImpl extends UserPropertyDescriptor {
-        public String getDisplayName() {
-            return null;    // not visible
```

Comment:
At least in the case of `UserPropertyDescriptor`, `User/configure.jelly` has

```xml
<j:forEach var="d" items="${descriptors}" varStatus="loop">
  <j:if test="${d.enabled}">
    <f:section title="${d.displayName}">
```

so the control is done via `isEnabled`, and if so `getDisplayName` is assumed to be nonnull.